src/server/models/Meter.js

```diff
@@ -229,6 +266,44 @@ class Meter {
 	readings(conn) {
 		return Reading.getAllByMeterID(this.id, conn);
 	}
+
+	/**
+	 * Makes the given meter valid.
+	 * @param {*} meter The meter.
+	 */
+	static makeMeterDataValid(meter) {
+		if (meter.unitId === -99) {
+			// If there is no unitId, set defaultGraphicUnit to null and displayable to false.
+			if (meter.defaultGraphicUnit !== -99) {
+				meter.defaultGraphicUnit = -99;
+				log.warn('defaultGraphicUnit has been removed since there is no unitId.');
```

Comment:
For this message and the similar ones that follow, I think more information should be included in the message. The log file does not have the context of the operation going on. Thus, I think the message should include the meter name.